server/data/licenceApiClient.ts

```diff
@@ -393,17 +393,17 @@ export default class LicenceApiClient extends RestClient {
     )) as LicenceEvent[]
   }
 
-  async getConditions(version: string): Promise<LicencePolicy> {
+  async getConditions(version: string): Promise<LicencePolicyResponse> {
     try {
-      return this.get({ path: `/licence-policy/version/${version}` }) as Promise<LicencePolicy>
+      return this.get({ path: `/licence-policy/version/${version}` }) as Promise<LicencePolicyResponse>
     } catch (error) {
       return error.status >= 400 && error.status < 500 ? null : error
     }
   }
 
-  async getActiveConditions(): Promise<LicencePolicy> {
+  async getActiveConditions(): Promise<LicencePolicyResponse> {
```

Comment:
would generally expect a method called getConditions to return a list of conditions.

maybe `getLicencePolicy` and `getLicencePolicyForVersion` would be better names for these?